pkg/common/args.go

```diff
@@ -48,6 +48,12 @@ func PrintInfo(clientSet *kubernetes.Clientset, config *rest.Config) {
 // ValidateArgs validates the arguments passed to the program
 // and creates the output directory if it doesn't exist
 
+func SetDefaultNamespace() {
+	if viper.Get("namespace") == "" {
+		viper.Set("namespace", DefaultNamespace)
+	}
+}
+
 func ValidateArgs() error {
 	if viper.Get("namespace") == "" {
```

Comment:
reuse SetDefaultNamespace() here too.

Reply:
Okay...

Reply:
> reuse SetDefaultNamespace() here too.

can you please elaborate this one? sorry but i not understood it..